plugin-core/src/main/java/appland/webviews/webserver/AppMapWebview.java

```diff
@@ -17,10 +18,10 @@
 @Getter
 public enum AppMapWebview {
     AppMap("appmap"),
-    InstallGuide("appland-install-guide"),
-    Findings("appland-findings"),
-    SignIn("appland-signin"),
-    Navie("appland-navie");
+    InstallGuide("install-guide"),
+    Findings("findings"),
+    Navie("navie"),
```

Comment:
nit: you could make this `navie.html` etc., this value is only used by `getIndexHtmlUrl()`, which appends `.html`. This would make it more clear that "navie" etc. is actually a filename.